src/components/thumbnail-card/ThumbnailCard.scss

```diff
@@ -86,3 +86,11 @@
         }
     }
 }
+
+.thumbnail-card-title-tooltip {
+    background-color: black;
```

Comment:
cc @booc0mtaco we probably don't want one-off tooltip styling?

Reply:
Correct, we also want to stick with our variables for colors instead of this CSS value.

if we want to define a tooltip style that is pure black (i am guessing this is the ask from design?), we should name that case and use the same param we do for callout (that navy blue color we sometimes see)

Reply:
So it looks like design has agreed that we should stick with the consistent tooltip. Will remove this block.